### PR TITLE
Feat/14/add something

### PR DESCRIPTION
**Pull request by Parenyuk**
add title

### CODE REVIEW

File: src/b1-ui/header/NavMenu.tsx [hunk 1]
```diff
@@ -9,9 +9,11 @@ export const NavMenu = () => {
     let {isAuth, avatar} = useSelector<AppStateType, InitialStateType>(state => state.loginPage)
     let dispatch = useDispatch();
 
+
```
Comment (TuzDmitry) [inline src/b1-ui/header/NavMenu.tsx:12]: What's the shit???

File: src/features/p2-RegPage/RegPage.tsx [hunk 2]
```diff
@@ -9,17 +9,15 @@ import {Preloader} from "../../b1-ui/common/preloader/Preloader";
...
+    const notification = useSelector<AppStateType, string>(state => state.regPage.notification);
+    const {isRegistrated, inProgress} = useSelector<AppStateType, InitialStateType>(state => state.regPage);
 
     const dispatch = useDispatch();
 
     const submitData = () => {
         dispatch(sendUserData(email, password))
     };
 
-
```
Comment (TuzDmitry) [inline src/features/p2-RegPage/RegPage.tsx:22]: could you plz delete one more string here?

File: src/b1-ui/header/NavMenu.tsx [hunk 3]
```diff
@@ -9,9 +9,11 @@ export const NavMenu = () => {
...
     let onLogOutClick = () => {
         dispatch(LogOut())
     };
+
```
Comment (TuzDmitry) [inline src/b1-ui/header/NavMenu.tsx:16]: good job!
i suugest you this code 
`<div>hello</div>`

File: src/b1-ui/header/NavMenu.tsx (hunk 1 above)
Comment (Parenyuk) [reply src/b1-ui/header/NavMenu.tsx:12]: @TuzDmitry  I need to do this!

File: src/features/p2-RegPage/RegPage.tsx (hunk 2 above)
Comment (Parenyuk) [reply src/features/p2-RegPage/RegPage.tsx:22]: I rewrite it.

**Review by TuzDmitry — APPROVED** [review]

**Pull request merged**